src/main/java/org/dependencytrack/policy/vulnerability/VulnerabilityPolicyProvider.java

```diff
@@ -21,4 +21,5 @@ public interface VulnerabilityPolicyProvider {
 
     PaginatedResult getAllVulnerabilityPolicies(AlpineRequest request);
 
+    VulnerabilityPolicy updateVulnerabilityPolicy(final org.dependencytrack.model.VulnerabilityPolicy updatedPolicy);
```

Comment:
We should not pass the database object here because any implementation of PolicyProvider should work